user/src/pages/index.tsx

```diff
@@ -1,5 +1,40 @@
-import { Typography } from "@material-ui/core";
+import "fontsource-roboto";
+
+import { Button, Typography } from "@material-ui/core";
+
+import styles from "../styles/pages/index.module.scss";
 
 export default function Home() {
-  return <Typography variant="h4">Hello, world :D!{"<->"}</Typography>;
+  return (
+    <div className={styles["structure"]}>
```

Comment:
If the class name doesn't have dashes ("-"), use dot notation instead. Example: `styles.structure`.